app/build.gradle.kts

```diff
@@ -47,6 +47,9 @@ dependencies {
     testImplementation(TestLibraries.jUnit)
     androidTestImplementation(TestLibraries.runnner)
     androidTestImplementation(TestLibraries.espressoCore)
+
+    //Third Party Libraries
```

Comment:
We could delete this.

Reply:
lets keep it.  there are other comments like this in this file.

Reply:
I think we can delete all comment in the file. naming is clear enough, we don't need to comments.